fastlane_core/lib/fastlane_core/env.rb

```diff
@@ -0,0 +1,12 @@
+module FastlaneCore
+  class Env
+    def self.enabled?(env)
+      return false unless ENV[env]
+      unless ENV["SKIP_ENV_HANDLER"]
```

Comment:
Is there a reason we need this? I mean, it's kind of weird that setting `ENV["SKIP_ENV_HANDLER"] = "0"` would actually disable this.